editor/plugins/spatial_editor_plugin.h

```diff
@@ -200,6 +200,7 @@ class SpatialEditorViewport : public Control {
 	Control *surface;
 	Viewport *viewport;
 	Camera *camera;
+	Camera *current_camera;
```

Comment:
Why save it as a member if you only need it in `_notification` where you set it?